pkg/documents/documents.go

```diff
@@ -101,43 +104,53 @@ func uploadCustomerDocument(logger log.Logger, repo DocumentRepository, bucketFa
 			return
 		}
 
-		file, _, err := r.FormFile("file")
-		if file == nil || err != nil {
+		// if r.Body is larger than maxFormSize an error will be returned when the body is read
+		r.Body = http.MaxBytesReader(w, r.Body, maxFormSize)
```

Comment:
TIL, nice find

Reply:
I know! I can't believe I hadn't seen it before.